scripts/main.js

```diff
@@ -0,0 +1,140 @@
+// Menu Button
+
+(() => {
+  const menu = document.querySelector('#menu');
+  const body = document.querySelector('body');
+  menu.addEventListener('click', () => {
+    const parent = document.createElement('div');
+    parent.classList.add('nav-menu', 'mobile');
+
+    parent.innerHTML = ` <div class="menu-header">
+    <img src="assets/icons/white-logo.svg">
+    <button id="closeMenu" type="button" class="closeButton"><img src="assets/icons/close.svg" alt=""></button>
+    </div>
+    <ul class="nav menu-links">
+    <li class="link">
+    <a rel="noopener" href="index.html"> Home </a>
+    </li>
+    <li class="link">
+    <a rel="noopener" href="about.html"> About </a>
+    </li>
+    <li class="link">
+    <a rel="noopener" href="index.html#main-program"> Program </a>
+    </li>
+    <li class="link">
+    <a rel="noopener" href="#"> Join </a>
+    </li>
+    <li class="link">
+    <a rel="noopener" href="#"> Sponsor </a>
+    </li>
+    <li class="link">
+    <a rel="noopener" href="#"> News </a>
+    </li>
+    <li class="link">
+    <button type="button" class="interaction interaction-alternative">CC Campaing</button>
+    </li>
+    </ul>`;
+
+    body.appendChild(parent);
+    const closeButton = document.querySelector('#closeMenu');
+    closeButton.addEventListener('click', () => {
+      body.removeChild(parent);
+    });
+
+    const links = document.querySelectorAll('.link');
+    links.forEach((link) => {
+      link.addEventListener('click', () => {
+        body.removeChild(parent);
+      });
+    });
+  });
+
+  const professors = [];
+
+  function professor(name, background, longDescription, image) {
+    return {
+      name,
+      background,
+      longDescription,
+      image,
+    };
+  }
+
+  // Professors
+
+  const names = ['Ranchoddas Shamalad Chanchad', 'Emilia Titor', 'David Spilbierg',
+    'Camilo Torres', 'Serana Titor', 'Hashimoto Kurasiki'];
+
+  const backgrounds = ['Berkman Professor of Institute of High Advanced Meaningful Development',
+    'Tech leader of CC Bootcamp Summer Foundation',
+    'Recognized developer of the most inteligent IA of the MIT',
+    '',
+    'Developer in charge of the Google API project for 2022',
+    'Berkman Professor of Tokyo Institute of Technology for the past 5 years'];
+
```

Comment:
- [ ] As per the requirement, you are expected to make an array of objects (speakers) and loop through the array to dynamically generate the speakers.
Something like this:point_down:
```
const speakers = [
  {
    name: "Julia Leda", 
    role: "President of Young pirates of Europe", 
    description: "European ingetration, political democracy, and participation of ...", 
    photo: "image_url_here"
  },
  {
    name: "Julia Leda2", 
    role: "President of Young pirates of Europe", 
    description: "European ingetration, political democracy, and participation of ...", 
    photo: "image_url_here"
  },
  ...
]
```
**Then you do this**
speakers.forEach(   We generate dynamically  )
Does that make sense?

Reply:
Hello @HENRYKC24 👋🏻 

Thanks for the overall review. I'll make the above changes to the project tomorrow. But this one is already implemented 

The array is defined here:
https://github.com/JohnFTitor/capstoneProject_first/blob/5b9e85dcc85817ee863cbf24ca3ce674ee0d1ede/scripts/main.js#L52

In this for loop, each object is created and added to the array.
As it was too much info, I made arrays for names, descriptions, and all to make a for loop to iterate through them and create the objects.

https://github.com/JohnFTitor/capstoneProject_first/blob/5b9e85dcc85817ee863cbf24ca3ce674ee0d1ede/scripts/main.js#L90-L94

And here I start iterating through each object in the renders function
https://github.com/JohnFTitor/capstoneProject_first/blob/5b9e85dcc85817ee863cbf24ca3ce674ee0d1ede/scripts/main.js#L98-L100

I did it this way because I wanted to use ```continue``` but linters gave me an error. I can change it for a ForEach since now I'm using an If to start the process. It would be better to read

Reply:
It will really be easier to read as you conclusively pointed out. Having the professors(speakers) in a bare array of objects(speakers) would greatly enhance the readability and I strongly advise you do that.

You are doing great I must say, @JohnFTitor. Make those changes and I am sure your project would be approved instantly. Keep up the good work:muscle:

Reply:
Thanks!